src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,103 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+    private static final int START_CAPACITY = 10;
+    private int sizeArray = 0;
```

Comment:
У нас int по дефолту має 0 , тому можна його не ініціалізувати.

```suggestion
    private int sizeArray;
```

Reply:
ok